### PR TITLE
zerotier-one: update to 1.8.4 and fix format

### PR DESCRIPTION
**Pull request by Nyovelt**
Topic Description
-----------------

zerotier-one: update to 1.8.4 and fix format

Package(s) Affected
-------------------

Security Update?
----------------

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [ ] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-network/zerotier-one/spec [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VER=1.8.3
+VER=1.8.4
 SRCS="tbl::https://github.com/zerotier/ZeroTierOne/archive/refs/tags/$VER.tar.gz"
-CHKSUMS="sha256::f0813900c812910db2751bfb43d22a06f11dbcfbe4515e6b24fcad90ca46f665"
+CHKSUMS="sha256::61b8c1ea5904cc87431939212033bb4d05d11f517860a01cac75f0090d94272b"
 CHKUPDATE="anitya::id=17578"
```
Comment (RedL0tus) [inline extra-network/zerotier-one/spec]: Missing new line at end of file. Otherwise LGTM.

**Review by RedL0tus — DISMISSED** [review]

File: extra-network/zerotier-one/spec (hunk 1 above)
Comment (WhiredPlanck) [inline extra-network/zerotier-one/spec]: Missing new line, otherwise LGTM too.

**Review by WhiredPlanck — DISMISSED** [review]

**Comment by WhiredPlanck** [comment]
@Nyovelt ping?

File: extra-network/zerotier-one/spec (hunk 1 above)
Comment (Nyovelt) [reply extra-network/zerotier-one/spec]: Fixed

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**